ddl/000018_staging_support_nil.up.sql

```diff
@@ -0,0 +1,4 @@
+BEGIN;
+ALTER TABLE kv_staging RENAME TO staging_kv;
```

Comment:
all ref tables in graveler (ddl 000017) are named graveler_<type>

maybe align the staging table as well?